docs/source/api/result_data.rst

```diff
@@ -1,12 +1,13 @@
 .. _ref_api_result_data:
 
-*****************
-ResultData object
-*****************
+**************
+``ResultData``
+**************
```

Comment:
```suggestion
*********************
``ResultData`` object
*********************
```

Reply:
Google Style Guide says to always follow a literal code string with a noun indicating what that code string is (file, class, method, object....)

Reply:
Correct, class. I've updated your comment.